opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -737,24 +708,25 @@ void EclipseWriter::writeTimeStep(int report_step,
 
         */
         {
-            Solution sol(restartHandle);
+            Solution sol(restartHandle);  //Type solution: confusing
 
             if (write_float)
                 sol.addFromCells<float>( cells );
             else
                 sol.addFromCells<double>( cells );
+            // MIssing a ENDSOL output.
             {
-                for (const auto& prop : simProps) {
-                    if (prop.enable_in_restart) {
-                        auto ecl_data = grid.compressedVector( prop.data );
-                        convertFromSiTo( ecl_data,
-                                         units,
-                                         prop.dim );
+                for (const auto& prop : cells) {
+                    if (prop.second.target == data::TargetType::RESTART_AUXILLARY) {
+                        auto ecl_data = grid.compressedVector( prop.second.data );
+                        //convertFromSiTo( ecl_data,
+                        //                 units,
+                        // prop.dim );
```

Comment:
why is this here?  Maybe better to add a written comment if there's an intention behind it.  Else remove.